taskcluster/ci/push-image/kind.yml

```diff
@@ -0,0 +1,59 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+
+loader: taskgraph.loader.transform:loader
+
+kind-dependencies:
+    - docker-image
+
+transforms:
+    - balrog_taskgraph.transforms.docker_push:transforms
+    - balrog_taskgraph.transforms.run_on_releases:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+task-defaults:
+    worker-type: images
+    worker:
+        taskcluster-proxy: true
+        docker-image: {in-tree: skopeo}
+        max-run-time: 3600
+    dependencies:
+          tests-js: ui-tests
+          build-js: ui-build
+          tests-backend: test-backend
+          tests-agent: test-agent
+    run-on-tasks-for: [github-release]
+    run-on-releases: [v]
```

Comment:
Why "v"? That seems particularly terse and impossible to search for.

Reply:
This is used as a regex, i.e. it means this task runs on release whose tag starts with the letter `v`, like today's .taskcluster.yml has `$if: 'tasks_for == "github-release" && release_tag[0] == "v"'`

Reply:
:face_palm

Of course!